nerfstudio/process_data/polycam_utils.py

```diff
@@ -216,7 +216,9 @@ def process_depth_maps(
 
     # Downscale depth maps
     summary_log.append(
-        process_data_utils.downscale_images(depth_dir, num_downscales, folder_name="depth", verbose=verbose)
+        process_data_utils.downscale_images(
+            depth_dir, num_downscales, folder_name="depths", nearest_neighbor=True, verbose=verbose
```

Comment:
NB: this is not only fixed to use nearest-neighbor, but I believe the `folder_name` was actually broken too?  it appears that `depth` is the 1x directory, and then `depths_{s}` is the dir for downscale by `s`